static/js/publisher/release/reducers/pendingReleases.js

```diff
@@ -154,6 +155,27 @@ function resumeProgressiveRelease(state, key) {
   return nextState;
 }
 
+function cancelProgressiveRelease(state, key, previousRevision) {
```

Comment:
Is this supposed to cancel all pending revisons with the same key?

What would happen if several revisions across different architectures (for example whole channel) was released progressively with the same key.
If `cancelProgressiveRelease` cancels all releases with the same key, will it cancel them all? How can only be just one `previousRevision`?

Reply:
I've added a comment above this function to explain a little